Source/WebCore/Modules/notifications/NotificationDataCocoa.mm

```diff
@@ -57,12 +57,12 @@
 
     String uuidString = dictionary[WebNotificationUUIDStringKey];
     auto uuid = UUID::parseVersion4(uuidString);
-    if (!uuid)
+    if (!uuid || !uuid->isValid())
```

Comment:
I do not think this is needed, 0 and 1 should not be allowed values for UUID Version4, ditto below.